### PR TITLE
Refactor Sylius RBAC initialize code to allow calling it in fixtures

### PR DESCRIPTION
**Pull request by stloyd**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| Fixes | #2513, #2550
| License       | MIT

### CODE REVIEW

**Comment by stloyd** [comment]
@pjedrzejewski @aRn0D ping :)

**Comment by askyoh** [comment]
Hello, as I commented in https://github.com/Sylius/Sylius/issues/2513, I still have a bug during new installation.

I read some of your posts, and so i did the command : sylius:rbac:initialize, and then, executed the sylius:install:setup again.

Then it worked.

Was it the good way to do it ?
fix will be implemented to initialize roles during installation ?

Thanks for reply!

**Comment by stloyd** [comment]
@askyoh Yes with this change it would always call rbac initialization, which would fix those issues.

Current code is broken in case you __don't__ use sample data while installing Sylius cause rbac initialization is used there only, which will lead to #2513 problems with missing data.

**Comment by aRn0D** [comment]
@pjedrzejewski can you handle that ?

File: src/Sylius/Bundle/RbacBundle/Doctrine/RbacInitializer.php [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        $this->roleRepository = $roleRepository;
+    }
+
+    protected function initialize(OutputInterface $output = null)
+    {
+        $this->initializePermissions($output);
+        $this->initializeRoles($output);
+    }
+
+    protected function initializePermissions(OutputInterface $output = null)
```
Comment (aRn0D) [inline src/Sylius/Bundle/RbacBundle/Doctrine/RbacInitializer.php]: I am not fan of that

**Comment by pjedrzejewski** [comment]
Nice, thanks a lot Joseph! I missed the time to do this. :)

**Comment by pjedrzejewski** [comment]
:+1:

**Comment by askyoh** [comment]
Hi again.

Juste updated the bundle, and now, i get this error :

---------
 Attempted to load class "RbacInitializer" from namespace "Sylius\Component\Rbac\Doctrine".
  Did you forget a "use" statement for "Sylius\Bundle\RbacBundle\Doctrine\RbacInitializer"?
---------

when i use install-database, or install:sample-data

**Comment by askyoh** [comment]
Hello, sorry to comment again on this topic, but I still get the error mentioned above.
I'm new on Sylius and beeing stuck during the installation is quite frustrating.

Besides, I found that in services.xml from RbacBundle, the RbacInitializer class is called from Sylius\Component\RbacBundle\Doctrine   while it is located in Sylius\Bundle\RbacBundle\Doctrine.

**Pull request merged**